Gems/ScriptCanvas/Code/Include/ScriptCanvas/Translation/GraphToLuaUtility.cpp

```diff
@@ -60,7 +60,7 @@ namespace GraphToLuaUtilityCpp
 
         for (;;)
         {
-            auto candidate = AZStd::string::format("]%s]", EqualSigns(numEqualSignsRequired).c_str());
+            auto candidate = AZStd::string::format("]%s", EqualSigns(numEqualSignsRequired).c_str());
```

Comment:
For testing, you are going to want to add a print node with a all of the normally escaped chars in C, like [Print Node "all \\ of this \/ stuff \. \' \" ]. I would also make sure there is a node with a string that just ends with "]", like "string with ]" from the original bug report.  You may want to consider making the function `MakeLongBracketString` public, and you can test it directly in the unit tests without making more graphs as the source of the function.   You can create a local `ScriptContext` object, and use it's `Execute` method in the test to make sure your print/build string argument nodes are still valid string arguments in Lua.